<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Uni-Frontend</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>321</height>
     </rect>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Resize">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>10</y>
      <width>33</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string notr="true">x4</string>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Icz">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dcz">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>9</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale Ratio:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddFile">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RemoveFile">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Run">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Append">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>This Context will be append to output file name.</string>
    </property>
    <property name="text">
     <string>out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>This Context will be append to output file name.</string>
    </property>
    <property name="text">
     <string>Appending:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Model:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Model">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Model</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Model">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearFile">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Information">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Backend:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Backend">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Backend">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Backend</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_tta">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable tta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LogCL">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
